nix/flakebox.nix

```diff
@@ -376,6 +376,15 @@ rec {
     '';
   };
 
+  # ciCrossTestAll = craneLibTests.mkCargoDerivation {
+  #   pname = "${commonCliTestArgs.pname}-all-cross";
```

Comment:
Taking a wild guess since this is commented out.

If you're bumping into issues running `nix build .#ci.ciCrossTestAll -L` due to cln socket errors, it's because the socket path > 108 chars when running locally.

https://unix.stackexchange.com/a/367012

I haven't thought of an elegant way to handle. I'll create an issue to track.